OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -830,6 +849,23 @@ static void ShowStatusBarsDropdown(DropDownButtonWidget dropdown, GameSettings s
 			dropdown.ShowDropDown("LABEL_DROPDOWN_TEMPLATE", 500, options.Keys, setupItem);
 		}
 
+		static void ShowDisplaySelectionDropdown(DropDownButtonWidget dropdown, GraphicSettings s)
+		{
+			var options = Exts.MakeArray(Game.Renderer.DisplayCount, i => "{0} {1}".F(DisplayBaseName, i + 1));
```

Comment:
Can we please change this method to
```csharp
static void ShowDisplaySelectionDropdown(DropDownButtonWidget dropdown, GraphicSettings s)
{
	Func<int, ScrollItemWidget, ScrollItemWidget> setupItem = (o, itemTemplate) =>
	{
		var item = ScrollItemWidget.Setup(itemTemplate,
			() => s.VideoDisplay == o,
			() => s.VideoDisplay = o);

		var label = "Display {0}".F(o + 1);
		item.Get<LabelWidget>("LABEL").GetText = () => label;
		return item;
	};

	dropdown.ShowDropDown("LABEL_DROPDOWN_TEMPLATE", 500, Enumerable.Range(0, Game.Renderer.DisplayCount), setupItem);
}
```

There are two motivations behind this request:
* There is no need to define the `options` array, and this just adds overhead resolving the array items
* The `DisplayBaseName` variable isn't needed, and hurts maintainability by introducing confusion about why it doesn't follow normal conventions more than it helps by reducing a duplicated label. All these hardcoded strings will eventually be replaced by a new translation system anyway.